services/wallet/router/sendtype/send_type.go

```diff
@@ -131,7 +111,11 @@ func (s SendType) canUseProcessor(p pathprocessor.PathProcessor) bool {
 	}
 }
 
-func (s SendType) processZeroAmountInProcessor(amountIn *big.Int, amountOut *big.Int, processorName string) bool {
+func (s SendType) SimpleTransfer(p pathprocessor.PathProcessor) bool {
```

Comment:
I think this is unused now. Moreover, we normally call these something like `isSimpleTransfer`

Reply:
@igor-sirotin I thought the linter checked unused functions and function parameters 🤔 did we disable that?

Reply:
Correct, after rebase, not needed.

Reply:
> @igor-sirotin I thought the linter checked unused functions and function parameters 🤔 did we disable that or did I just make that up?

Indeed I think it was there. We need to review linter checks and ensure it's there 👍